src/wallet/rpcwallet.cpp

```diff
@@ -3253,25 +3257,27 @@ UniValue z_getaddressforaccount(const UniValue& params, bool fHelp)
         return NullUniValue;
     if (fHelp || params.size() < 1 || params.size() > 3)
         throw runtime_error(
-            "z_getaddressforaccount account ( [\"pool\", ...] diversifier_index )\n"
+            "z_getaddressforaccount account ( [\"receiver_type\", ...] diversifier_index )\n"
             "\nFor the given account number, derives a Unified Address in accordance"
             "\nwith the remaining arguments:\n"
-            "\n- If no list of pools is given (or the empty list \"[]\"), the best and"
-            "\n  second-best shielded pools, along with the transparent pool, will be used."
+            "\n- If no list of receiver types is given (or the empty list \"[]\"), the best"
+            "\n  and second-best shielded receiver types, along with the \"transparent\" (i.e. P2PKH) receiver"
```

Comment:
The receiver type is `p2pkh`.
```suggestion
            "\n  and second-best shielded receiver types, along with the \"p2pkh\" (i.e. transparent) receiver"
```